features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/utils/ImageViewUtils.kt

```diff
@@ -0,0 +1,221 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.utils
+
+import android.graphics.Rect
+import android.graphics.drawable.Drawable
+import android.view.View
+import android.widget.ImageView
+import com.datadog.android.sessionreplay.internal.recorder.densityNormalized
+import com.datadog.android.sessionreplay.model.MobileSegment
+
+internal class ImageViewUtils {
```

Comment:
> [!NOTE]
> This class has no instance property, does it really need to exist as a class to be instantiated? It could be an Object, or just top level util functions?

Reply:
I've changed this to an object, since as top level functions it makes injecting them for unit tests more complicated